bioconda_utils/lint_functions.py

```diff
@@ -225,8 +225,10 @@ def has_windows_bat_file(recipe, metas, df):
 @lint_multiple_metas
 def should_be_noarch(recipe, meta, df):
     deps = _get_deps(meta)
+    no_compilers = sum(1 for i in deps if
+                        i.startswith('toolchain_')) == 0
```

Comment:
We'll have to make this one a bit more generic in the future when the Anaconda built compilers are used.
A similarly hardcoded but more general solution could be
```python
    no_compilers = all(
        not dep.startswith(('clang_', 'clangxx_', 'gcc_', 'gxx_', 'gfortran_', 'toolchain_'))
        for dep in deps)
```

Reply:
done, thanks.